shaders/framebuffer/renderframebuffer.frag

```diff
@@ -25,14 +25,25 @@
 #include "floatoperations.glsl"
 #include <#{fragmentPath}>
 
+#define exposure #{rendererData.hdrExposure}
+#define DeltaError 0.013f
+#define MaxValueColorBuffer 1E10
+
 layout(location = 0) out vec4 _out_color_;
 layout(location = 1) out vec4 gPosition;
 layout(location = 2) out vec4 gNormal;
+layout(location = 3) out vec4 filterBuffer;
 
 void main() {
-     Fragment f   = getFragment();
-     _out_color_  = f.color;
-     gPosition    = f.gPosition;
-     gNormal      = f.gNormal;
-     gl_FragDepth = normalizeFloat(f.depth);
+    Fragment f  = getFragment();
+    _out_color_ = vec4((log2(vec3(1.0) - (f.color.rgb - vec3(DeltaError)))/(-exposure)), f.color.a);
```

Comment:
Is this the inverse of the tone mapping? In that case, I guess it is only the inverse of one of the TMOs - in that case, which one?

I thought the idea was to do this only for non-hdr fragments, but this seems to be happening for all fragments (?)

My interpretation of this is that all fragments are now supposed to be rendered in "LDR" (non-linear), get transformed to "HDR" (where they can be added linearly) and then get mapped back? I think this calls for some discussion. @jccosta when are you available to talk?

What is `DeltaError`?